kitsune/sumo/jinja2/includes/common_macros.html

```diff
@@ -6,7 +6,7 @@
         <input type="hidden" name="{{ k }}" value="{{ v }}" />
       {% endfor %}
     {% endif %}
-    <input type="search" name="q" required="required" aria-required="true" placeholder="{{ _('Find help...')|truncate(12) }}" class="searchbox" id="search-q" />
+    <input type="search" name="q" required="required" aria-required="true" placeholder="{{ _('Find help...')|truncate(12) }}" class="searchbox" id="{{id+'shivani'}}" />
```

Comment:
A couple of problems with this. 

1. `id` is not necessarily a string. you can see in the macro signature that it's `False` by default.
2. adding your name to the ID of an element on the site is not appropriate.

Reply:
@pmac Hi
Thanks for the review.
>1. id is not necessarily a string. you can see in the macro signature that it's False by default.

I thought id would be different for each, hence, wanted to incorporate into the code. I'll try to add it in a different way :)
>2. adding your name to the ID of an element on the site is not appropriate.

I wanted to include a random string instead of "search" as this error may arise again due to same id. I'll remove my name, please suggest me a set of characters that would be good?